jiant/scripts/postproc/glue/README.md

```diff
@@ -0,0 +1,13 @@
+# GLUE Benchmark Submission Formatter
+
+`jiant` supports generating submission files for [GLUE](https://gluebenchmark.com/). To generate test predictions, use the `--write_test_preds` flag in [`runscript.py`](https://github.com/jiant-dev/jiant/blob/master/jiant/proj/main/runscript.py) when running your workflow. This will generate a `test_preds.p` file in the specified output directory. To convert `test_preds.p` to the required GLUE submission format, use the following command:
+
+```bash
+python glue_submission_formatter.py --input_base_path $INPUT_BASE_PATH --output_path $OUTPUT_BASE PATH
+```
+
+where `$INPUT_BASE_PATH` contains the task folder(s) output by [runscript.py](https://github.com/jiant-dev/jiant/blob/master/jiant/proj/main/runscript.py). Alternatively, a subset of tasks can be formatted using:
+
+```bash
+python glue_submission_formatter.py --tasks cola mrpc --input_base_path $INPUT_BASE_PATH --output_path $OUTPUT_BASE PATH
+```
```

Comment:
I'll update this doc in a subsequent PR to reflect the benchmark script changes.

Reply:
Ah, okay. I think it's cause my PR moves the file, but I guess the merged `origin/master` deletes it. Should I do anything about this currently?

Reply:
Feel free to leave it as is. We may move things around when we clean up the repo in ~2 weeks. This can be part of that.